lib/modules/dosomething/dosomething_signup/includes/SignupTransformer.php

```diff
@@ -85,7 +85,11 @@ protected function transform($item) {
       $campaign = (object) $item->campaign;
       $current_run = $campaign->campaign_runs['current']['en']['id'];
       $current = ($item->campaign_run == $current_run);
-      $data += $this->transformSignup($item, $current);
+      $campaign_run = node_load($current_run);
+      $current_run_start = $campaign_run->field_run_date['en'][0]['value'];
+      $current_run_end = $campaign_run->field_run_date['en'][0]['value2'];
```

Comment:
@aaronschachter oh shoot - I didn't mean for this to be apart of this PR - I thought I switched branches but looks like I didn't! This is meant for #6166 . I'm going to delete this stuff and just have changes made in `dosomething_signup.query.inc.